proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -42,16 +42,18 @@ message EventCreateBatch {
   string project_location = 7;
 }
 
-// EventReceive is an event emitted when credits are received either upon
-// creation of a new batch or upon transfer. Each batch_denom created or
-// transferred will result in a separate EventReceive for easy indexing.
+// EventReceive is an event emitted when credits are received either via
+// creation of a new batch, transfer of credits, or taking credits from a
+// basket. Each batch_denom created, transferred or taken from a baset will
+// result in a separate EventReceive for easy indexing.
 message EventReceive {
+
   // sender is the sender of the credits in the case that this event is the
   // result of a transfer. It will not be set when credits are received at
-  // initial issuance.
+  // initial issuance or taken from a basket.
```

Comment:
would it be useful to have an enum distinguishing the sender type? I assume we'd put the basket denom in sender, but maybe consumers would want the enum so they know what to do with the string in `sender`?

Reply:
could you make that update @technicallyty ? although I'd say let's not overload `sender`, but instead add `basket_denom`. We don't want consumers to choke on a denom instead of address in `sender`

Reply:
done, and ran proto-gen